components/contribution-flow/ContributionFlowSuccess.js

```diff
@@ -304,7 +346,7 @@ class ContributionFlowSuccess extends React.Component {
               </Flex>
             </ContainerWithImage>
             <Flex flexDirection="column" alignItems="center" justifyContent="center" width={1}>
-              {isPendingBankTransfer ? this.renderBankTransferInformation() : this.renderCallsToAction()}
+              {this.addSuccessInfoByPaymentMethod()}
```

Comment:
For me, a good name would start with `render`

Reply:
Makes sense, I've renamed it to `renderInfoByPaymentMethod`; maybe that looks better. 🤔